.travis.yml

```diff
@@ -9,20 +9,26 @@ php:
 before_install:
   - pip install --user cpp-coveralls
 
+env:
+  - CC=clang
+  - CC=gcc
+
 script:
   - cd src
   - phpize
   - ./configure --enable-snuffleupagus --enable-coverage
-  - make -j 2
+  - make
```

Comment:
Why remove -j2 ? (better yet, use nproc or /proc/cpuinfo to get the number of cores)